tensilelite/Tensile/SolutionStructs.py

```diff
@@ -3451,6 +3459,19 @@ def subCheckLdsBlockSizePerPad(tc, idx):
           ldsBiasMaxElements = max(ldsBiasMaxElements, state["MacroTile0"] * dataType.numBytes())
       ldsNumBytes = max(ldsNumBytes, state["LdsOffsetBias"] + ldsBiasMaxElements)
 
+    state["LdsBytesNoAmax"] = ldsNumBytes
+    if state["ProblemType"]["OutputAmaxD"]:
+      if state["GlobalSplitU"] > 1:
+        reject(state, "OutputAmaxD doesn't support GlobalSplitU > 1 yet")
```

Comment:
shouldn't check it here.
our GSU solution is enabled by runtime's GPU kernel argument.
The rejection should happen in runtime predicate.

Reply:
@jichangjichang  Yes, I also added the predicate in Contraction.py and ContrationProblemPredicates.hpp. The rejection here is just another guard to avoid generating the code.

Reply:
No, What I mean is that you shouldn't reject for generating the code. The kernel can still support GSU > 1 with AmaxD == nullptr.

Reply:
done